Src/Support/Google.Apis.Auth.AspNetCore/Google.Apis.Auth.AspNetCore.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <PropertyGroup>
+    <!-- Only ASP.NET Core 2.0 on netstandard2.0 supported -->
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Beta release -->
+    <Version>$(Version)-beta01</Version>
```

Comment:
Is this how versions are generally represented in this repo? After release, would this follow the same version number as Google.Apis.Auth?

Reply:
We haven't used beta releases in this repo for a very long time, so there isn't a "normal" way to do this.
`$(Version)` is the current support library version; so right now this version will be `1.36.1-beta01`.
I anticipate that we'll release betas following the support library version (i.e. if we upgrade to support library version 1.37.0, then this package will be released as `1.37.0-beta01`), and incrementing the `beta` number as required.
Then this `<Version>` line will be removed to release as GA.